src/ctelnet.cpp

```diff
@@ -1914,7 +1914,12 @@ void cTelnet::handle_socket_signal_readyRead()
         }
         else
         {
-            if( ch != '\r' && ch != 0 ) cleandata += ch;
+            if( ch == TN_BELL )
+            {
+                // flash taskbar for 3 seconds
+                QApplication::alert( mudlet::self(), 3000 );   
+            }
+            else if( ch != '\r' && ch != 0 ) cleandata += ch;
```

Comment:
Do not make this an `else if` leave it as an `if` so that, yes it does flash the toolbar as you wish - but _it still gets put into the data that gets passed into the rest of the system_

🤔  It would be better from an API point of view to have a Lua command to do the `QApplication::alert( mudlet::self(),`\<user argument number of seconds>`);` then they can detect the bell character (as any other) and process it as they see fit (play a "knock-knock" or other sound for instance instead) or indeed flash the toolbar if that is their wish.

@vadi2 Your thoughts?